cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -63,6 +67,7 @@ def import_handler(request, course_key_string):
         json: import a course via the .tar.gz file specified in request.FILES
     """
     course_key = CourseKey.from_string(course_key_string)
+    library = isinstance(course_key, LibraryLocator)
```

Comment:
It seems like it might be better to extract the common functionality into a single function, and then pass in the small number of values that depend on `library`, since they're mostly filenames and such. That would make it clear what the code that varies is, vs the code that's common between courses and libraries.